services/api-server/.env-devel

```diff
@@ -13,6 +13,7 @@
 #
 
 API_SERVER_DEV_FEATURES_ENABLED=1
+LOG_FORMAT_LOCAL_DEV_ENABLED=1
```

Comment:
I would advise agains this one. If you require logs in the `full format` just add the env var in the project's root folder `.env` file.
This file is also captured by the osparc autoteplyer and interpolated when it composes the env vars. That's why I would not add it here. Let's avoid unwanted behaviours.

Reply:
- this .env-devel is only use for testing/development in ``api-server` only. NOTE that this is NOT the .env-devel of the base!
- `LOG_FORMAT_LOCAL_DEV_ENABLED` was introduced about a month ago after refactoring logs system (@matusdrobuliak66)

Reply:
yes